static/js/services/live-list.js

```diff
@@ -22,6 +15,19 @@ angular.module('inboxServices').factory('LiveListConfig',
     'use strict';
     'ngInject';
 
+    var HTML_BIND_REGEX = /ng-bind-html="([^"]+)"([^>]*>)/gi;
```

Comment:
Possibly more correct, although I don't know the implications of passing an empty string to the parser:
```
/(^|\s)ng-bind-html="([^"]*)"([^>]*>)/gi
```
But you'd need to add some whitespace if a linebreak were removed.